README.md

```diff
@@ -2,18 +2,113 @@
 
 ## Pre-requisite
 
-- Kubernetes or OpenShift cluster running (ex: minikube / minishift)
-- Istio installed
+- Kubernetes or OpenShift cluster running (ex: minikube 0.27+ / minishift)
+- Istio 0.8+ installed 
 - For visualization support, Kiali installed
 
+## Prepare
+
+Clone the project and set the env **DEMO_MESH_ARENA_HOME**
+```bash
+cd ~/software
+git clone https://github.com/jotak/demo-mesh-arena.git
+export DEMO_MESH_ARENA_HOME=/Users/nicolas/software/demo-mesh-arena
+```
+
 ## Build & Run for Kubernetes
 
-(to be completed)
+Provision a local K8S cluster with **minikube**
+
+```bash
+minikube start \
```

Comment:
Just one quick question: are these minikube command line parameters mandatory? Or just `minikube start` could be enough?